lms/envs/aws.py

```diff
@@ -696,6 +705,9 @@
 FIELD_OVERRIDE_PROVIDERS += (
```

Comment:
Why do we need both a `FIELD_OVERRIDE_PROVIDER` and `MODULESTORE_FIELD_OVERRIDE_PROVIDERS` pointing at the save `SelfPacedDateOverrideProvider`?